lightning/src/ln/channelmanager.rs

```diff
@@ -1104,6 +1104,37 @@ impl ChannelDetails {
 	}
 }
 
+/// Used by [`ChannelManager::list_pending_payments`] to express the status of payments that are
+/// currently pending resolution. These not only include payments that are currently inflight, but
+/// also those that may have yet to find a successful path or have outstanding HTLCs.
```

Comment:
How is "outstanding" defined here? I can see it being read as "in-flight" but that would be redundant with the first part of the sentence. More generally, we should be clear and consistent throughout all the docs and identifiers with these terms, possibly eliminating redundant ones:
- pending
- resolved / unresolved / resolution
- inflight
- outstanding
- tracked
- recent
- fulfilled
- delivery

Both related to HTLCs and payments.